src/shared/ui/survey/MediaItems/PhotoItem.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC } from 'react';
+
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+
+import {
+  colors,
+  GALLERY_PHOTO_OPTIONS,
+  PHOTO_TAKE_OPTIONS,
+  requestCameraPermissions,
+  requestGalleryPermissions,
+  useCameraPermissions,
+  useGalleryPermissions,
+} from '@shared/lib';
+import { PhotoIcon } from '@shared/ui';
+
+import MediaInput from './MediaInput';
+import MediaValue from './types';
+
+type Props = {
+  onChange: (value: MediaValue) => void;
+  value?: MediaValue;
+};
+
+const PhotoItem: FC<Props> = ({ onChange, value }) => {
+  const { isCameraAccessGranted } = useCameraPermissions();
+  const { isGalleryAccessGranted } = useGalleryPermissions();
+
+  const handlePickImage = (response: any) => {
+    onChange(response);
+  };
```

Comment:
You don't need this function.
Just use `onChange` on `launchImageLibrary` and `launchCamera` function calls